Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/ViewSettingsController.java

```diff
@@ -73,28 +74,38 @@
             "org:resource",
             "org:role"
     );
+    
+    private String[] measures = {
+            "case_count",
+            "case_relative",
+            "total",
+            "mean",
+            "median",
+            "max",
+            "min"
+    };
```

Comment:
If this is a closed set, perhaps it'd work well as an enum.

Reply:
This is only private array used in this specific ViewSettingsController class, not elsewhere, so it's not a big deal to bring it up as a separate class or enum.

Reply:
Another point is the string values are the UI keys, e.g. list box item keys.